invenio_audit_logs/services/service.py

```diff
@@ -88,7 +70,7 @@ def read(
         **kwargs,
     ):
         """Read a record."""
-        self.require_permission(identity, "read", user_identity=identity)
+        self.require_permission(identity, "read")
```

Comment:
why removing the injection of the param?

Reply:
@sakshamarora1 can you answer, based on what we discussed yesterday?

Reply:
We have removed the `CurrentUser` permission, it will be system identity which creates the logs now, hence the removal of this param.